src/equations/2d/compressible_euler_pot.jl

```diff
@@ -0,0 +1,681 @@
+@doc raw"""
+    CompressibleMoistPotTempEulerEquations2D
+
+The compressible Euler equations for a  gas with dry air and water vapor and a condensed phase 
+in two space dimensions.
+"""
+struct CompressibleMoistPotTempEulerEquations2D <: AbstractCompressibleEulerEquations{2, 6}
```

Comment:
If I recall our conversation correctly, @OsKnoth, this system of equations was supposed to be just the Euler equations with potential temperature and dry air. However, it seems like this is virtually identical to the moist equations. Or am I misunderstanding something here?